### ISSUE TITLE
lang bug in sidecar export window?

### ISSUE THREAD

**Issue by mrtngrsbch** [hvdwolf/jExifToolGUI#56]
The sidecar export windows are displayed in English and the strings are translated into /jExifToolGUI/translations/program_strings_es_ES.properties

Video:
[ScreenShot 2020-08-19 a las 10.03.58.mp4.zip](https://github.com/hvdwolf/jExifToolGUI/files/5096834/ScreenShot.2020-08-19.a.las.10.03.58.mp4.zip)

![ScreenShot 2020-08-19 a las 10 49 04](https://user-images.githubusercontent.com/693328/90643215-b788cb80-e209-11ea-993f-ea5cbaef1665.jpg)

best

ExifToolGUI 1.5.0
OSX 10.15.6

**Comment by hvdwolf** [comment]
I never received that translation from you. I sent a mail on Thursday 13 about "Again new translations"  and my ideas for 1.6. You mentioned in a mail from  "fr 14 aug. 01:24"

"Hi Harry,

1. I've just pulled these changes
2. Very interesting these new options for 1.6 !  I'm going to test what wonders we can do with them
"

But I never received these translations, so I used Google translate to translate the menu and left the rest for what it was and then released on Saturday evening.
